src/serenity/default_vars.hpp

```diff
@@ -11,7 +11,13 @@ namespace qos_pipeline {
 const constexpr char* VALVE_OPENED = "VALVE_OPENED";
 constexpr bool DEFAULT_VALVE_OPENED = true;
 const constexpr char* ENABLED_VISUALISATION = "ENABLED_VISUALISATION";
-constexpr bool DEFAULT_ENABLED_VISUALISATION = true;
+constexpr bool DEFAULT_ENABLED_VISUALISATION = false;
+
+const constexpr char* ON_EMPTY_CORRECTION_INTERVAL =
```

Comment:
Why didn't you put those in pipeline class?
Also, this name does not explains what it do.